test/parallel/test-repl-persistent-history.js

```diff
@@ -178,6 +179,21 @@ const tests = [
     test: [UP],
     expected: [prompt]
   },
+  {
+    before: function before() {
+      if (!common.isWindows) {
+        try {
+          fs.symlinkSync('/dev/null', devNullHistoryPath);
+        } catch (ex) {
+          if (ex.code !== 'EEXIST')
+            throw ex;
+        }
+      }
+    },
+    env: { NODE_REPL_HISTORY: devNullHistoryPath },
```

Comment:
No, because I believe Windows would then fail since /dev/null does not exists there and currently there is no way to conditionally skip a test from running, you can only make it fall back to the empty history file scenario.